patches/http-server@14.1.1.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/lib/core/index.js b/lib/core/index.js
+index 920e55b9c01222c280b7f629ab4c8aaac8a987f7..a2464e449d59cf184ff89ac53f048639b541310e 100644
+--- a/lib/core/index.js
++++ b/lib/core/index.js
+@@ -319,6 +319,8 @@ module.exports = function createMiddleware(_dir, _options) {
+       res.setHeader('content-length', stat.size);
+       res.setHeader('content-type', contentType);
+ 
++      res.setHeader('Origin-Agent-Cluster', '?0');
```

Comment:
we tweak the http-server that we use to drive the tests to permit it to consider the localhost on a different realm as part of the same origin. This is the new way that chrome expects you to allow that. Despite doing this, you still cannot directly manipulate the iframe's document (this just eliminates one barrier around that), hence the `postMessage` communication in the realm-dom-test.js